modules/team_planner/spec/support/onboarding/onboarding_steps.rb

```diff
@@ -29,22 +29,27 @@
 module OnboardingSteps
   def step_through_onboarding_team_planner_tour
     next_button.click
+    wait_for_reload
     expect(page).to have_text sanitize_string(I18n.t('js.onboarding.steps.team_planner.overview')), normalize_ws: true
 
     # The team planner is long to load
     next_button.click
+    wait_for_reload
     expect(page)
-      .to have_text sanitize_string(I18n.t('js.onboarding.steps.team_planner.calendar')), normalize_ws: true, wait: 10
+      .to have_text sanitize_string(I18n.t('js.onboarding.steps.team_planner.calendar')), normalize_ws: true, wait: 20
```

Comment:
In b5b5b5bb (funny commit sha btw), I already raised timeout from 5 to 10, and now we're raising it from 10 to 20.

10 seconds seems to be already a lot of time. It's fine to raise it to 20, but then if it fails again sometimes, we'll have to search for a fix in another direction.

Reply:
Waiting for 20 seconds for anything feels already reeeeeally excessive. Are we really expecting this JS to not finish before 20 seconds?

Reply:
I know it's a lot. I don't really know why this takes so long to load and it's definitely worth addressing but until then I'd prefer to wait 10 more seconds than to have to re-run the whole suite because this spec failed just because it wanted a bit more time. Do you have any ideas why the JS takes so long?